alvr/common/src/sockets/stream_socket/mod.rs

```diff
@@ -254,6 +269,7 @@ impl StreamSocket {
         client_ip: IpAddr,
         port: u16,
         protocol: SocketProtocol,
+        byterate: u32,
```

Comment:
To be extra clear I would rename this to video_byterate, since the total byterate is calculated.

Reply:
Done. I also propagated the change to ThrottlingSettings, as it has the same meaning.